### PR TITLE
Add xeora-cli 1.0.17506

### PR DESCRIPTION
**Pull request by freakmaxi**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/xeora-cli.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'xeora-cli' do
+  version '1.0.17506'
+  sha256 :no_check
```
Comment (commitay) [inline Casks/xeora-cli.rb]: Please add the `sha256`

File: Casks/xeora-cli.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  name 'Xeora CLI'
+  homepage 'http://www.xeora.org/'
+
+  depends_on cask: 'dotnet-sdk'
+
+  binary 'xeora'
+
+  preflight do
+    FileUtils.mv "#{staged_path}/xeora", "#{staged_path}/xeora-cli"
+    File.open("#{staged_path}/xeora", 'w') { |f| f.write("#!/bin/sh\nXEORAPATH=#{staged_path} #{staged_path}/xeora-cli $@\n") }
```
Comment (commitay) [inline Casks/xeora-cli.rb]: Why is the preflight needed?

**Review by commitay — DISMISSED** [review]
Please rebase on master to fix the CI error.

File: Casks/xeora-cli.rb (hunk 2 above)
Comment (freakmaxi) [reply Casks/xeora-cli.rb]: I need to prepare XEORAPATH variable according to the installation directory. so I thought I can do it in preflight. Should I do it at postflight? or should I use another way?

Comment (commitay) [reply Casks/xeora-cli.rb]: a `shimscript` is preferred
https://github.com/caskroom/homebrew-cask/blob/0b54d5b3a4f1761198c3aa4df29627693e11ef33/Casks/logisim-evolution.rb#L13-L23

File: Casks/xeora-cli.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+cask 'xeora-cli' do
+  version '1.0.17506'
```
[the remark this answers is not in the document]
Comment (freakmaxi) [reply Casks/xeora-cli.rb]: No there is not any changing log yet. That latest version is linked to the version 1.0.17506. when there is a new release, this latest will be linked to that. Currently, we have only one version which is 1.0.17506. When it has a changelog, I'll create a new pull request with `appcast`.

**Review by commitay — APPROVED** [review]

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**